src/adapter/src/catalog.rs

```diff
@@ -2504,14 +2503,11 @@ impl Catalog {
 
                         // Update the catalog storage and Builtin Tables.
```

Comment:
Might want to update this comment since it's not 100% accurate. We're not updating the catalog storage yet.

Reply:
Good call, updated!